.github/workflows/ci.yml

```diff
@@ -35,11 +35,8 @@ jobs:
           pixi-version: v0.39.0
           cache: true
           environments: lint
-      - name: Run Pylint, Mypy & Pyright
-        run: |
-          pixi run -e lint pylint
-          pixi run -e lint mypy
-          pixi run -e lint pyright
```

Comment:
Can't have pyright AND mypy in the same project. They _will_ eventually result conflicting with each other, and in fact they did so heavily in #53.
Chose mypy as I found pyright extremely obnoxious and factually wrong in too many cases.

Reply:
> Can't have pyright AND mypy in the same project. They will eventually result conflicting with each other, and in fact they did so heavily in https://github.com/data-apis/array-api-extra/pull/53.

This isn't true at all, and all of my typed projects are proof of this.
In my experience, there are indeed situations where mypy and pyright behave differently. In at least 99% of these cases, this is caused by one of the >1200 confirmed bugs in mypy that are present in pyright: https://github.com/erictraut/mypy_issues

> Chose mypy as I found pyright extremely obnoxious and factually wrong in too many cases.

Do you have an example of this?